File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int n=x;
        int s=0;
        while(n>0)
        {
          int  k=n%10;
            s=s*10+k;
            n=n/10;
        }
        if(x==s){
            return true;
        }
        else
        return false;
        
    }
}

File: 9-palindrome-number/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let copynum=x, rev=0;
    while(copynum>0){
        const last=copynum%10;
        rev=rev*10+last;
        copynum=Math.floor(copynum/10);

    }
    return x===rev;;
    
};